atlassian/bitbucket/cloud/repositories/repositoryVariables.py

```diff
@@ -61,7 +61,7 @@ def each(self, q=None, sort=None):
 
         return
 
-    def get(self, uuid):
+    def get(self, uuid: str):  # type: ignore[override]
```

Comment:
The signature for `get` in `AtlassianRestAPI` doesn't match here.  So we need to override.